src/slack_interaction_handler.ts

```diff
@@ -464,9 +464,10 @@ export async function handleCommandUnclaimed(
       arg = arg.substr(1); // strip off the # prefix
     }
     if (!(arg in slackChannelIds)) {
-      await SlackApiUtil.sendMessage(`Channel #${arg} not found`, {
-        channel: channelId,
```

Comment:
wasn't the issue also that channelId was actually a slack user id in one case? did you figure out how that happened? and can you change the variable name, or before assigning it ensure that it is a channel id?

Reply:
I've lost the original thread with the error message .. do you have a link?

Reply:
https://voteamericahelpline.slack.com/archives/G01DNHJ3AM7/p1604253054010300?thread_ts=1604250068.007300&cid=G01DNHJ3AM7

Reply:
It looks like (1) they passed an `@someuser` argument, and (2) slack is configured to escape the arguments.. for this command it should not be. updated the slack config doc to reflect that.

Reply:
So there were multiple issues here -- misconfigured slack but also code bugs?

Reply:
user error + misconfigured slack.  this change is fine

Reply:
Is there a way to make the code resilient to the user error?

Reply:
I'm going to approve this -- if it's still not resilient to user error can you fix or create an issue so we can get back to it at some point?

Reply:
(And sorry if it is and I'm not following correctly)

Reply:
I think we're good.  The error message makes it clear we expected a channel name, and turning off escaping will make it not look weird.